tests/nominal/test_GroupRareLevelsTransformer.py

```diff
@@ -81,8 +85,9 @@ class TestFit(GenericFitTests, WeightColumnFitMixinTests, DummyWeightColumnMixin
     def setup_class(cls):
         cls.transformer_name = "GroupRareLevelsTransformer"
 
+    @pytest.mark.parametrize("from_json", [True, False])
     @pytest.mark.parametrize("library", ["pandas", "polars"])
-    def test_learnt_values_no_weight(self, library):
+    def test_learnt_values_no_weight(self, library, from_json):
```

Comment:
can remove this as it's a `fit` test
```suggestion
    def test_learnt_values_no_weight(self, library):
```